### PR TITLE
New skrell emotes

### PR DESCRIPTION
**Pull request by LocoTokyoFunBus**
Adds more funny frog sounds, of my own creation

### CODE REVIEW

File: code/modules/emotes/definitions/_species.dm [hunk 1]
```diff
@@ -88,9 +88,11 @@
 
 /datum/species/skrell
 	default_emotes = list(
-		/decl/emote/audible/warble
+		/decl/emote/audible/warble,
+		/decl/emote/audible/croon,
+		/decl/emote/audible/lowarble,
+		/decl/emote/audible/croak,
```
Comment (Alberyk) [inline code/modules/emotes/definitions/_species.dm:94]:
```suggestion
		/decl/emote/audible/croak
```

File: code/modules/emotes/definitions/audible.dm [hunk 2]
```diff
@@ -255,3 +255,18 @@
...
+
+/decl/emote/audible/croon
+	key = "croon"
+	emote_message_3p = "USER croons..."
+	emote_sound = list('sound/voice/croon1.ogg', 'sound/voice/croon2.ogg')
+
+/decl/emote/audible/lowarble
+	key = "lwarble"
+	emote_message_3p = "USER lets out a low, throaty warble!"
+	emote_sound = 'sound/voice/lowarble.ogg'
```
Comment (Alberyk) [inline code/modules/emotes/definitions/audible.dm:267]:
```suggestion
	emote_sound = 'sound/voice/low warble.ogg'
```

**Review by Alberyk — CHANGES_REQUESTED** [review]
The lwarble sound file is named lowarble in the code, while the file itself is named low warble. You can either change it in the code or change the file name.

**Comment by LocoTokyoFunBus** [comment]
!review

**Review by Alberyk — APPROVED** [review]

**Review by Casper3667 — APPROVED** [review]

**Pull request merged**